src/Datadog.Trace.ClrProfiler.Managed/Integrations/HttpClientIntegration.cs

```diff
@@ -0,0 +1,772 @@
+using System;
+using System.IO;
+using System.Linq;
+using System.Net.Http;
+using System.Threading;
+using System.Threading.Tasks;
+using Datadog.Trace.ClrProfiler.Emit;
+using Datadog.Trace.Logging;
+
+namespace Datadog.Trace.ClrProfiler.Integrations
+{
+    /// <summary>
+    /// Tracer integration for HttpClient.
+    /// </summary>
+    public static class HttpClientIntegration
+    {
+        private const string IntegrationName = "HttpClient";
+        private const string SystemNetHttp = "System.Net.Http";
+        private const string Major4 = "4";
+        private const string HttpClientTarget = "System.Net.Http.HttpClient";
+        private const string DeleteAsync = "DeleteAsync";
+        private const string SendAsync = "SendAsync";
+        private const string GetAsync = "GetAsync";
+        private const string GetByteArrayAsync = "GetByteArrayAsync";
+        private const string GetStreamAsync = "GetStreamAsync";
+        private const string GetStringAsync = "GetStringAsync";
+        private const string PostAsync = "PostAsync";
+        private const string PutAsync = "PutAsync";
+        private static readonly Vendors.Serilog.ILogger Log = DatadogLogging.GetLogger(typeof(HttpClientIntegration));
+
+        /// <summary>
+        /// Instrumentation wrapper for HttpClient.GetStringAsync.
+        /// </summary>
+        /// <param name="handler">The HttpClient instance to instrument.</param>
+        /// <param name="uri">The <see cref="Uri"/> that represents the current request uri.</param>
+        /// <param name="opCode">The OpCode used in the original method call.</param>
+        /// <param name="mdToken">The mdToken of the original method call.</param>
+        /// <param name="moduleVersionPtr">A pointer to the module version GUID.</param>
+        /// <returns>Returns the value returned by the inner method call.</returns>
+        [InterceptMethod(
+#pragma warning restore CS0419 // Ambiguous reference in cref attribute
```

Comment:
What "ambiguous reference" is this referring to? Is it something we can fix instead of ignoring the warning?

Reply:
Actually, this line is restoring the warning, but it is never disabled above. We can remove it.
```suggestion
```

Reply:
Removing.